app/services/folio/locations_map.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Folio
+  # This starts with the location.tsv file provided by libsys. We make any modifications to that data here.
+  # This is intended to be a temporary class, which will only be necessary until we complete the Folio migration.
+  class LocationsMap
```

Comment:
```suggestion
  # This is intended to be a temporary class, which will only be necessary until we complete the Folio migration.
  # TODO: remove after migration
  class LocationsMap
```
makes it easier to find later?